src/components/CallView/shared/LocalVideo.vue

```diff
@@ -411,9 +411,14 @@ export default {
 }
 
 .bad-connection-quality {
+	/* This adds black bars around the video depending on the resolution,
+	 * but it's still better than having your half-transparent avatar
+	 * shine an underlying video/avatar through it. */
+	background-color: #000000;
+
 	.video,
 	.avatar-container {
-		opacity: 0.5
+		opacity: 0.3
```

Comment:
I would keep opacity to 0.5, as that value was set for consistency with [the opacity of remote videos during an ICE disconnection](https://github.com/nextcloud/spreed/blob/master/src/components/CallView/CallView.vue#L531-L537).